proto/sifnode/ethbridge/v1/query.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+package sifnode.ethbridge.v1;
+
+import "gogoproto/gogo.proto";
+import "sifnode/ethbridge/v1/types.proto";
+import "sifnode/oracle/v1/types.proto";
+
+option go_package = "github.com/Sifchain/sifnode/x/ethbridge/types";
+
+// Query service for queries
+service Query {
+  // EthProphecy queries an EthProphecy
+  rpc EthProphecy(QueryEthProphecyRequest) returns (QueryEthProphecyResponse) {}
+}
+
+// TODO check rest property names
```

Comment:
Let's do some cleanup if we merge it in as is so we don't have to do it when it gets bigger

Reply:
Is this todo carried over from the previous query?

Reply:
I've added it as there was one field name distinction in the json tag, will update that in the Proto now.